### PR TITLE
Added logic to start import after cluster creation

### PR DESCRIPTION
**Pull request by shtripat**
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

File: tendrl/commons/flows/create_cluster/__init__.py [hunk 1]
```diff
@@ -61,6 +62,22 @@ def run(self):
         # SSH setup jobs finished above, now install ceph
         ceph_help.create_ceph(self.parameters)
 
+        # Start jobs for importing cluster
+        node_list.remove(NS.node_context.node_id)
+        new_params = self.parameters.copy()
```
Comment (shtripat) [inline tendrl/commons/flows/create_cluster/__init__.py]: Sure. Will add them.

**Comment by r0h4n** [comment]
Please update this PR and resolve merge conflicts, I have merged your Gluster create cluster PR here https://github.com/Tendrl/commons/pull/254/files

**Comment by r0h4n** [comment]
Still required after https://github.com/Tendrl/commons/pull/316 ?

**Comment by shtripat** [comment]
@r0h4n this is not required now. Closing it.

**Pull request closed without merging**